pyproject.toml

```diff
@@ -54,7 +54,7 @@ ipaddr = { version = "^2.2.0", optional = true }
 ipfabric = { version = "~6.0.9", optional = true }
 ipfabric-diagrams = { version = "~6.0.2", optional = true }
 isodate =  { version = "^0.6.1", optional = true }
-meraki = { version = "^1.7.2", optional = true }
+meraki = { version = "<=1.45.0", optional = true }
```

Comment:
Want a minimum version here still I assume?

Reply:
Ah, good point. I will add in.